src/components/addBook.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuid } from 'uuid';
+import { addBook } from '../redux/books/books';
+
+const AddBook = () => {
+  const dispatch = useDispatch();
+
+  const submitBookToStore = (e) => {
+    e.preventDefault();
+    const form = document.getElementsByTagName('form')[0];
+    const newBook = {
+      id: uuid(),
+      title: form.firstChild.value,
+    };
+    dispatch(addBook(newBook));
+  };
```

Comment:
The component responsible for adding a new book should contain:
- [ ] - A local state with  title, author. I'd suggest using useState hook.
- [ ] - Input field should make use of (onChange) listen to update local state.